packages/@uppy/box/types/index.test-d.ts

```diff
@@ -1,2 +1,13 @@
 import Box = require('../')
-// TODO implement
+import Uppy = require('@uppy/core')
+
+{
+ const uppy = Uppy()
```

Comment:
can we already use `new Uppy()` in these? since that will be the only way in 2.0. i don't remember for sure if our current typings allow it.